src/main/java/com/rtbhouse/kafka/workers/impl/record/weigher/ByteArrayWeigher.java

```diff
@@ -0,0 +1,16 @@
+package com.rtbhouse.kafka.workers.impl.record.weigher;
+
+import com.rtbhouse.kafka.workers.api.record.Weigher;
+
+public class ByteArrayWeigher implements Weigher<byte[]> {
```

Comment:
Sample/default implementations seem to be part of user API so they should be part of api package.

Reply:
Done